### PR TITLE
refactor: add app key to s3 apps workflows

### LINKED ISSUE #37: Add app key to apps workflows

### PR DESCRIPTION
**Pull request by pyphilia**
close #37

### CODE REVIEW

File: .github/workflows/cdelivery-s3-apps.yml [hunk 1]
```diff
@@ -36,6 +36,8 @@ on:
       # Environment variables
       app-id:
         required: true
```
Comment (spaenleh) [inline .github/workflows/cdelivery-s3-apps.yml:38]: Will we need to remove this then ?

File: .github/workflows/cdelivery-s3-apps.yml [hunk 2]
```diff
@@ -81,9 +83,9 @@ jobs:
       # Set environment variables required to perform the build. These are only available to this step
       env:
         REACT_APP_GRAASP_DOMAIN: ${{ secrets.graasp-domain }}
-        REACT_APP_GRAASP_APP_ID: ${{ secrets.app-id }}
+        REACT_APP_GRAASP_APP_KEY: ${{ secrets.app-key }}
         REACT_APP_SENTRY_DSN: ${{ secrets.sentry-dsn }}
-        REACT_APP_MOCK_API: false
+        REACT_APP_ENABLE_MOCK_API: false
```
Comment (spaenleh) [inline .github/workflows/cdelivery-s3-apps.yml:88]: Nice, did we update the docs of the apps-query-client and/or of the apps to reflect this change ?

**Review by spaenleh — APPROVED** [review]

File: .github/workflows/cdelivery-s3-apps.yml (hunk 1 above)
Comment (pyphilia) [reply .github/workflows/cdelivery-s3-apps.yml:38]: We actually still need the app id to deploy the app on aws.

**Comment by pyphilia** [comment]
We potentially don't need to merge this PR because most of the apps don't use the workflow to be deployed.

**Comment by spaenleh** [comment]
@pyphilia should we close this ? or merge it anyway ?

**Pull request closed without merging**